frontend/src/components/Students/StudentsPage.js

```diff
@@ -43,11 +41,10 @@ export const StudentsPage = (props) => {
                 students={students}
                 setStudent={setStudent}
                 selectStudentSuccess={selectStudentSuccess}
-                updateStudentSuccess={updateStudentSuccess}
                 setIsOpenUpdateDialog={setIsOpenUpdateDialog}
                 setIsOpenConfirmDialog={setIsOpenConfirmDialog}
                 checkAllStudentsSuccess={checkAllStudentsSuccess}
-                {...rest}
+                {...props}
```

Comment:
If you give to components props then you don't need give him students Students, selectStudentSuccess and td... Because this function and this data we already have in props

Reply:
done